### ISSUE TITLE
seekToDefaultPosition doesn't start the playback from the live position or live edge

### ISSUE THREAD

**Issue by avenger-8** [androidx/media#1777]
### Version

Media3 1.3.1

### Devices that reproduce the issue

Pixel 8 Pro running Android 14

### Reproducible in the demo app?

Not tested

### Reproduction steps

1. Play a live audio stream
2. Pause the live stream
3. Play it again

### Expected result

The playback position should be in line with the live 'head' of the stream when paused. So, when the playback is resumed it plays from the live posiiton and not from the position it was paused.

`fun pause() {
        player.pause()
    }`

`fun play() {
        if (isLive) {
            player.seekToDefaultPosition()
            player.prepare()
        }
        player.play()
    }`

### Actual result

The live stream audio resumes from the point it was paused.

### Media

The issue is not dependent on a particular Media item. It needs to be a live audio stream for it to be reproducible.

### Bug Report

- [X] You will email the zip file produced by `adb bugreport` to android-media-github@google.com after filing this issue.

**Comment by icbaker** [comment]
>  It needs to be a live audio stream for it to be reproducible.

Please provide a link to such a stream that we can use to reproduce (otherwise our interpretation of this might be different to you, which will waste time trying to repro with the wrong setup). Please either post it here or send it to android-media-github@google.com with the subject `Issue #1777`. Please also update this issue to indicate you’ve done this.

**Comment by avenger-8** [comment]
I have sent the livestream url to the email as suggested above. Thank you.

**Comment by icbaker** [comment]
I'm not able to reproduce the behaviour you observe. I used the following code at the bottom of [`PlayerActivity.initializePlayer()`](https://github.com/androidx/media/blob/c35a9d62baec57118ea898e271ac66819399649b/demos/main/src/main/java/androidx/media3/demo/main/PlayerActivity.java#L298) in the main demo app (without the `prepare()` call inside `play()` from your example, I don't think that's needed?):

```
Handler handler = Util.createHandlerForCurrentLooper();
handler.postDelayed(
    () -> {
      player.pause();
      handler.postDelayed(
          () -> {
            player.seekToDefaultPosition();
            player.play();
          },
          10_000);
    },
    5000);
```

This code waits for 5s after the start of playback then pauses. Then it waits 10s before seeking to the live edge and resuming playback. When playing your stream with this code in place, I observe that playback **does not** resume in the same place it was paused (as you report).

Please can you try this same code in our demo app with your stream, and see what you observe?

**Comment by avenger-8** [comment]
Unfortunately, this doesn't work for us. The live stream resumes in the same place after it's paused. Do we need to set LiveConfiguration by any chance for this to work or configure ExoPLayer so that it doesn't buffer live content and always plays from the live edge?

**Comment by avenger-8** [comment]
Just tested based on what you had suggested before. It was actually an issue in our custom ForwardingPlayer class where we were removing the COMMAND_SEEK_TO_DEFAULT_POSITION command incorrectly for the live stream. All good now.

**Comment by avenger-8** [comment]
I just noticed an issue hence I have reopened it. We can have this code with the Player play function and it works as expected and plays from the live edge as you have suggested:

`player.seekToDefaultPosition()`

However, this cannot be achieved via the Play/Pause buttons given on the Playback notification that is shown out of the box by Media3. It plays the media from the point it was paused.

<img width="252" alt="notif" src="https://github.com/user-attachments/assets/7602e9df-3958-4299-838f-61339a28d349">


How do we modify the notification controls to play from the live edge for live streams?
